packages/forms/src/directives/validators.ts

```diff
@@ -13,7 +13,11 @@ import {AbstractControl} from '../model';
 import {emailValidator, maxLengthValidator, maxValidator, minLengthValidator, minValidator, NG_VALIDATORS, nullValidator, patternValidator, requiredTrueValidator, requiredValidator} from '../validators';
 
 
-/**
+export function toInteger(value: string|number): number {
+  return typeof value === 'number' ? value : parseInt(value, 10);
+}
+
+/**s
```

Comment:
Looks like an accidental change.
```suggestion
/**
```